src/modules/jcms_notifications/src/NotificationService.php

```diff
@@ -0,0 +1,62 @@
+<?php
+
+namespace Drupal\jcms_notifications;
+
+use Aws\AwsClientInterface;
+use Aws\Sns\SnsClient;
+use Drupal\Core\Site\Settings;
+
+/**
+ * Class NotificationService.
+ *
+ * @package Drupal\jcms_notifications
+ */
+class NotificationService {
+
+  /**
+   * @var \Aws\Sns\SnsClient
+   */
+  protected $snsClient;
+
+  protected $endpoint = '';
+
+  protected $topicName = '';
+
+  protected $region = '';
+
+  /**
+   * NotificationService constructor.
+   *
+   * @param \Aws\AwsClientInterface|NULL $sns_client
+   */
+  public function __construct(AwsClientInterface $sns_client = NULL) {
+    $this->endpoint = Settings::get('jcms_sqs_endpoint');
+    $this->region = Settings::get('jcms_sqs_region');
+    $this->topicName = Settings::get('jcms_sns_topic');
```

Comment:
there are actually many different topics, one per content type (e.g. `bus-blog-articles--end2end`, `bus-podcast-episodes-end2end`):
https://github.com/elifesciences/builder/blob/master/projects/elife.yaml#L464
so configuring one string is not enough, but to avoid configuring all of them (which is boring) we could configure a template like `bus-%s--end2end` where you can substitute the content type you need.